### ISSUE TITLE
Duplicated commands

### ISSUE THREAD

**Issue by AMiller42** [Vyxal/Vyxal#516]
This is a list of all commands/overloads that are duplicated. Boxes will be checked when the dupe is resolved, either upon removal, or upon deciding to keep the dupe.

- [ ] `ɾ` (string: a) / `⇧` (string: a) - `a.uppercase()`
- [ ] `ɽ` (string: a) / `⇩` (string: a) - `a.lowercase()`
- [ ] `E` (num: a) / `Ǎ` (num: a) - `2 ** a`
- [ ] `ʀ` (string: a) / `∞` (any: a) / `ṁ` (string: a) - `palindromised a`

**Comment by chunkybanana** [comment]
`ṁ` does *vertical* mirror, `ʀ`'s pointless since`∞` works on lists.

**Comment by chunkybanana** [comment]
`⇧`/`⇩` don't vectorise so `ɾ`/`ɽ` are better sometimes

**Comment by cgccuser** [comment]
Any suggestions for replacing `ʀ`?

**Comment by pxeger** [comment]
`ʁ` could be prefixes including the empty string? (and that could also apply to lists) That kinda mirrors its behaviour on integers as well

**Comment by AMiller42** [comment]
> `ṁ` does _vertical_ mirror

No, `øṁ` does vertical mirror, whereas `ṁ` does palindromise: [Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJgSGVsbG9cbldvcmxkYOG5gSIsIiIsIiJd)

Also, `øṁ` doesn't actually appear to do anything, which is an entirely different issue.
